commandline/WSL/install_guide.md

```diff
@@ -51,9 +51,9 @@ Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-L
 ```
 
 ### After enabling Windows Subsystem for Linux
-**Reboot when prompted**
+**Restart when prompted**

```

Comment:
It's just a Microsoft Style thing. The style guide is here, and you can search for "reboot" for the exact guidance. https://worldready.cloudapp.net/Styleguide/Read?id=2700